### DISCUSSION TITLE
Not Able to run any command After cypress run command in Docker

### DISCUSSION THREAD

**Discussion by JUPUDI-VAMSI-KALYAN** [cypress-io/cypress#21330] [Docker]
`FROM cypress/included:9.5.4
RUN mkdir /app
WORKDIR /app
COPY . /app
RUN npm install
RUN $(npm bin)/cypress verify
USER root
ENTRYPOINT []
CMD node testData/downloader.js \
    && node testData/converter.ts \
    && npm run cy:run \ 
    && npm run uploader`

First three commands in cypress are working file but the last command is not working. Any way to run this command in docker.

**Comment by d-lyx** [comment]
I have the same issue
